src/tasks/appveyor.js

```diff
@@ -41,7 +41,7 @@ module.exports = (config) => {
         'npm install',
       ],
       before_test: [
-        'cmd: npm install webpack@^%webpack_version%',
+        'cmd: npm install webpack@%webpack_version%',
```

Comment:
The `^`  should be equal to 'install the lastest version available within the current semver `major` range' (all `minor` && `patch` releases). Why is it needed to drop it ? 🙃